### ISSUE TITLE
WebUI/Talk: failed to render: template: views/talk:79:29

### ISSUE THREAD

**Issue by Nold360** [mudler/LocalAI#2691]
**LocalAI version:** 
quay.io/go-skynet/local-ai:v2.18.0-cublas-cuda12-ffmpeg

**Environment, CPU architecture, OS, and Version:**

Kubernetes 1.29

**Describe the bug**
I randomly klicked "Talk" on the webui and it gives me this error:

```json
{"error":{"code":500,"message":"failed to render: template: views/talk:79:29: executing \"views/talk\" at \u003c.Name\u003e: can't evaluate field Name in type schema.OpenAIModel","type":""}}
```

**To Reproduce**
Install 2.18 i guess

**Expected behavior**

**Logs**

**Additional context**
Pretty sure it was working in 2.17

**Comment by mudler** [comment]
yup confirm this - should be fixed by https://github.com/mudler/LocalAI/pull/2685

**Comment by mudler** [comment]
Closing as fixed in v2.18.1
